aion/test/test_helper.exs

```diff
@@ -1,19 +1,3 @@
 ExUnit.start
```

Comment:
I guess this file is not needed anymore?

Reply:
It is
```ex
19:52:22.599 [error] Task #PID<0.314.0> started from #PID<0.73.0> terminating
** (stop) exited in: GenServer.call(ExUnit.Server, {:take_async_cases, 16}, 60000)
    ** (EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started

```